### ISSUE TITLE
The contrast ratio of the "Run" icon is less than 3:1 in query editor

### ISSUE THREAD

**Issue by v-ruizh** [microsoft/vscode-cosmosdb#2435]
**OS:** Windows 10
**Build Version:** 20241120.1
**Regression:** Not a Regression

**Repro Steps:**
1. Create a "Cosmos DB for NoSQL" account.
2. Create a database and a container for the above account.
3. Expand the above account -> Right click the "Documents" node -> Select "Open Query Editor (preview)".
4. Check whether the contrast ratio of the "Run" icon is greater than 3:1 in query editor.

**Expect:**
The contrast ratio of the "Run" icon is greater than 3:1 in query editor.

**Actual:**
The contrast ratio of the "Run" icon is less than 3:1 in query editor. (Set VS Code to **Light theme**)
![Image](https://github.com/user-attachments/assets/80d5561d-e696-4bbb-ab89-dddffc7145de)
![Image](https://github.com/user-attachments/assets/4f31da10-c5e0-4d8a-bd1f-a78c21365489)

**More Info:**
This issue also **reproduces** in **Dark theme**.
![Image](https://github.com/user-attachments/assets/0b59e755-b504-4078-bad0-55233f969aaa)
![Image](https://github.com/user-attachments/assets/4c5e762f-cd8c-4619-9680-0430c3582fe6)

**Comment by bk201-** [comment]
@v-ruizh  As I understand your point, the contrast ration should be greater than 3:1, right?

Could you please share the app name what you did use for measurements :)

**Comment by v-ruizh** [comment]
> [@v-ruizh](https://github.com/v-ruizh) I've fixed. For dark theme the contrast ratio is 7.1:1. However, for light theme the contrast ratio is 4,5:1, One test out 6 didn't passed, since CCA recommends to use 7:1. Does it still an issue?

3:1 is enough for Non-text.
